src/main/java/frc/robot/RobotContainer.java

```diff
@@ -279,42 +306,71 @@ private Command speakerSource() {
 
   private Command intakeSpeaker() {
     return Commands.parallel(
-        intakeCommandFactory.intakeSpeakerCommandSmart(shooterCommandFactory.getBeamBreak()),
-        triggerCommandFactory.getGroundLoadCommand(shooterCommandFactory.getBeamBreak()));
+        intakeCommandFactory.getSpeakerIntakeCommand(), triggerCommandFactory.getLoadCommand());
   }
 
   private Command intakeAmp() {
     return Commands.parallel(
             trapElvCommandFactory.intakeGround(), intakeCommandFactory.getAmpIntakeCommand())
         .until(trapElvCommandFactory.getSourceBreak())
-        .andThen(trapElvCommandFactory.intakeFromGroundForTime());
+        .andThen(trapElvCommandFactory.intakeFromGroundForTime())
+        .asProxy();
   }
 
   private Command shootSpeaker() {
     return Commands.parallel(
-        Commands.either(
-            triggerCommandFactory.getShootCommand(),
-            triggerCommandFactory
-                .getShootCommand()
-                .onlyIf(() -> shooterCommandFactory.isShooterReady())
-                .asProxy(),
-            OI.getButton(OI.Operator.simple)),
+        Commands.waitSeconds(CommandConstants.WAIT_FOR_TRAPELV)
```

Comment:
What are we waiting for here?  Is this to make sure that isShooterReady is stable?  If so, we should use a debounce instead

Reply:
we are waiting for the trapelv to move out of the way